static/css/community.css

```diff
@@ -394,8 +394,40 @@ body.cid-community #cncf-code-of-conduct h2:after {
   body.cid-community a.community-cta-button {
     font-size: 1.5rem;
   }
+
+  body.cid-community .community-section h2:before,
+body.cid-community .community-section h2:after {
+  background-color: #aaaaaa;
+  content: "";
+  display: inline-block;
+  height: 1px;
+  position: relative;
+  vertical-align: middle;
+  width: 18%;
+}
+
+
 }
 
+@media only screen and (max-width: 440px) {
+
+  body.cid-community .community-section h2:before,
+body.cid-community .community-section h2:after {
+  background-color: #aaaaaa;
+  content: "";
+  display: inline-block;
+  height: 1px;
+  position: relative;
+  vertical-align: middle;
+  width: 7%;
+}
```

Comment:
Some further indentation would help improve readability, agree?

Reply:
At lines 408 and 409 there are empty lines , these lines can be dropped.